CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Grafana Mimir
 
+* [ENHANCEMENT] OTLP: Add support for converting OTel Exponential Histograms to Prometheus Native Histograms. #4063
```

Comment:
Ingestion of native histograms must also be enabled in ingesters. I think it's worth mentioning here too.